generator/namespaces.py

```diff
@@ -84,35 +101,21 @@ def namespaces(custom_namespaces, generated_sql_uri, app_listings_uri):
     with urllib.request.urlopen(app_listings_uri) as f:
         # groupby requires input be sorted by key to produce one result per key
         app_listings = sorted(json.loads(gzip.decompress(f.read())), key=get_app_name)
-    view_definitions = _get_views(generated_sql_uri)
+    db_views = _get_db_views(generated_sql_uri)
     namespaces = {}
     for app_name, group in groupby(app_listings, get_app_name):
-        views = defaultdict(list)
+        group = list(group)
         canonical_app_name = None
```

Comment:
I think this should reuse the pattern from https://github.com/mozilla/lookml-generator/blob/5a702a7d385ab155e8c4a52c6ea8df84a292d0fd/generator/lookml.py#L149-L153

```suggestion
        # use canonical_app_name where channel=="release" or the first one
        canonical_app_name = next(
            (app for app in group if app.get("channel") == "release"),
            group[0],
        )["canonical_app_name"]
```

Reply:
Much better